vitrina/datasets/urls.py

```diff
@@ -196,10 +196,15 @@
         name="dataset-rdf-download",
     ),
     path(
-        "datasets/<int:pk>/structure/import/",
+        "datasets/<int:pk>/version/<int:version_id>/structure/import/",
         DatasetStructureImportView.as_view(),
         name="dataset-structure-import",
     ),
+    path(
+        "datasets/<int:pk>/structure/import/",
```

Comment:
Does this mean, that this url is for the latest one?

(not a suggestion to change, just trying to get to know what this is about)

```suggestion
        "datasets/<int:pk>/version/latest/structure/import/",
```

Reply:
When you go and create the structure for the first time there is no version created. So its not latest, the version does not exist yet